frontend/app/services/TouchpointBackend.scala

```diff
@@ -111,6 +113,12 @@ object TouchpointBackend {
   lazy val Normal = TouchpointBackend(BackendType.Default)
   lazy val TestUser = TouchpointBackend(BackendType.Testing)
 
+  val f = Future {
```

Comment:
`val f` is unused, so you can just remove that - no needs a reference to this right now, unless you want to somehow tie this into the healthcheck?